.github/workflows/buildPreview.yml

```diff
@@ -0,0 +1,56 @@
+name: Compile extension PR preview
+
+on:
+  pull_request:
+    paths:
+      - "./extension/**"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Build extension
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+      - run: yarn install && yarn build:extension-preview
+      - name: Zip extension build
+        uses: montudor/action-zip@v0.1.0
+        with:
+          args: zip -qq -r ./extension/build/build.zip ./extension/build
+      - name: Delete Release # if it exists
```

Comment:
delete the old version of the PR preview so we can replace it.

this fails the first time, but this fails gracefully so it does not stop this pipeline